js/impress.js

```diff
@@ -724,24 +724,31 @@
         //   Well, the [tab] key by default navigates around focusable elements, so clicking
         //   it very often caused scrolling to focused element and breaking impress.js
         //   positioning. I didn't want to just prevent this default action, so I used [tab]
-        //   as another way to moving to next step... And yes, I know that for the sake of
-        //   consistency I should add [shift+tab] as opposite action...
+        //   as another way to moving to next step...
         document.addEventListener( "keyup", function( event ) {
 
+            // Handles Tab and Shift + Tab
+            if ( event.keyCode === 9 ) {
+              if ( event.shiftKey ) {
+                api.prev();
+              } else {
+                api.next();
+              }
+              return;
+            }
+
             if ( event.shiftKey || event.altKey || event.ctrlKey || event.metaKey ) {
```

Comment:
It was reported couple of times: https://github.com/impress/impress.js/issues?utf8=%E2%9C%93&q=is%3Aissue+alt+tab

It seems it doesn't happen to everyone, so may be system and/or browser dependent.